web/src/settings_bots.ts

```diff
@@ -125,6 +125,16 @@ function sort_bot_email(a: BotInfo, b: BotInfo): number {
 }
 
 function sort_bot_owner(a: BotInfo, b: BotInfo): number {
+    if (a.no_owner && b.no_owner) {
```

Comment:
Would be good to add a comment mentioning that we want to show bots without owner at bottom.